src/openms/source/CHEMISTRY/ModificationsDB.cpp

```diff
@@ -344,26 +323,13 @@ namespace OpenMS
 
           for (vector<String>::iterator orig_it = origins.begin(); orig_it != origins.end(); ++orig_it)
           {
-            if (orig_it->size() == 1)
+            if ((orig_it->size() == 1) && (*orig_it != "B") && (*orig_it != "J") && (*orig_it != "Z"))
```

Comment:
There is only one case where it applies: "MOD:00306" which has origin "J" (represents "I or L", so isn't really a modification anyway). We used to read it, but I assume it never got used (why look for modifications of residue J?). Now it's skipped.